pkg/function/data_test.go

```diff
@@ -99,6 +101,18 @@ func (s *DataSuite) TearDownSuite(c *check.C) {
 	}
 }
 
+func (s *DataSuite) ObjStoreDataCleanUp(c *check.C) {
+	if c.Failed() {
+		ctx := context.Background()
+		profile := s.profile
+		if localMinio, ok := os.LookupEnv("LOCAL_MINIO"); ok && localMinio == "true" {
```

Comment:
is the local minio ("cluster") instance started on each job run?
does local minio need to be cleaned up?